### PR TITLE
COMMERCE-5605 Adding an Account from Minium Account Management page is not working

### PR DESCRIPTION
**Pull request by riccardo-alberti**

### CODE REVIEW

**Comment by riccardo-alberti** [comment]
ci:test:sf

**Comment by riccardo-alberti** [comment]
ci:test:relevant

**Comment by riccardo-alberti** [comment]
hey @drewbrokke could you please check if the change is ok? thx

File: modules/apps/commerce/commerce-account-admin-web/src/main/java/com/liferay/commerce/account/admin/web/internal/portlet/action/EditCommerceAccountMVCActionCommand.java [hunk 1]
```diff
@@ -249,6 +249,7 @@ private ServiceContext _getServiceContext(
 
 		serviceContext.setCompanyId(_portal.getCompanyId(actionRequest));
 		serviceContext.setUserId(_portal.getUserId(actionRequest));
+		serviceContext.setScopeGroupId(_portal.getScopeGroupId(actionRequest));
```
Comment (drewbrokke) [inline modules/apps/commerce/commerce-account-admin-web/src/main/java/com/liferay/commerce/account/admin/web/internal/portlet/action/EditCommerceAccountMVCActionCommand.java:253]: SF: this should be sorted alphabetically

File: modules/apps/commerce/commerce-account-web/src/main/java/com/liferay/commerce/account/web/internal/portlet/action/EditCommerceAccountMVCActionCommand.java [hunk 2]
```diff
@@ -248,6 +248,7 @@ private ServiceContext _getServiceContext(
 
 		serviceContext.setCompanyId(_portal.getCompanyId(actionRequest));
 		serviceContext.setUserId(_portal.getUserId(actionRequest));
+		serviceContext.setScopeGroupId(_portal.getScopeGroupId(actionRequest));
```
Comment (drewbrokke) [inline modules/apps/commerce/commerce-account-web/src/main/java/com/liferay/commerce/account/web/internal/portlet/action/EditCommerceAccountMVCActionCommand.java:251]: SF: this should be sorted alphabetically

**Comment by drewbrokke** [comment]
@riccardo-alberti just the SF comments but otherwise looks good 👍

**Comment by riccardo-alberti** [comment]
@drewbrokke thanks a lot!!

**Comment by riccardo-alberti** [comment]
ci:test:sf

**Comment by riccardo-alberti** [comment]
ci:test:relevant

**Comment by riccardo-alberti** [comment]
ci:foward

**Comment by riccardo-alberti** [comment]
ci:forward

**Pull request closed without merging**